tensorboard/proto/README.md

```diff
@@ -0,0 +1,15 @@
+# TensorFlow Protos
```

Comment:
Could we actually move the proto directory under compat?  I think making this the top-level "proto" directory could be misleading, since TensorBoard does have some protos it defines itself (e.g. for plugin metadata), whereas these are actually all originally defined by TensorFlow, and at least for now TensorFlow will continue to be the source of truth.  This way they're scoped to just being present for compatibility purposes (and if TensorBoard ever becomes the source-of-truth we can change it then).